### PR TITLE
T18 - Armstrong Number

### PR DESCRIPTION
**Pull request by Ajay07Metamation**

### CODE REVIEW

File: Training/Program.cs [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
+using static System.Console;
+Write ("Enter a number to check for Armstrong Number : ");
+if (int.TryParse (ReadLine (), out int num) && num > 0)
+   GetAmstrong (num);
+else WriteLine ("Enter Valid Number");
+void GetAmstrong (int n) {
+   int exp = n.ToString ().Length;
+   double res = 0;
+   while (n > 0) {
+      res += Math.Pow ((n % 10), exp);
```
Comment (MetaNaveen) [inline Training/Program.cs:11]: Can you avoid inbuilt function Math.Pow()?

File: Training/Program.cs [hunk 2]
```diff
@@ -1,7 +1,15 @@
...
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿//Program to check for Amstrong Number
+using static System.Console;
+Write ("Enter a number to check for Armstrong Number : ");
+if (int.TryParse (ReadLine (), out int num) && num > 0)
+   GetAmstrong (num);
```
Comment (MetaNaveen) [inline Training/Program.cs:5]: GetAmstrong() implies something is returned.
Can you make the method to return true if it is an Armstrong number.

By the way, it's Armstrong number, not Amstrong.

File: Training/Program.cs [hunk 3]
```diff
@@ -1,7 +1,15 @@
...
+else WriteLine ("Enter Valid Number");
+void GetAmstrong (int n) {
+   int exp = n.ToString ().Length;
+   double res = 0;
+   while (n > 0) {
+      res += Math.Pow ((n % 10), exp);
+      n /= 10;
+   }
+   WriteLine ($"It is {((res == num) ? "" : "not ")}an Armstrong Number");
```
Comment (MetaNaveen) [inline Training/Program.cs:14]: You are comparing double against integer. Are you sure it's safe to compare double against integer?

**Review by MetaNaveen — CHANGES_REQUESTED** [review]

File: Training/Program.cs (hunk 3 above)
Comment (Ajay07Metamation) [reply Training/Program.cs]: All changes and formatting done in T18

**Comment by MetaNaveen** [comment]
**Final Notes For Consideration:**

- **return (res == num) ? true : false;** can be simplified into **return res == num;**

**Pull request closed without merging**